Typo fixes in readme

richlv opened the pull request.

daodennis-ri commented:
@richlv can you also add to your PR another spelling/grammar change?
On line 281 here: https://github.com/aws-samples/inspector2-enablement-with-cli/blob/main/inspector2_enablement_with_awscli.sh#L281

`# Because there is a loop on each account, this code is not subject to throttling API issue: only ONE account is process at the tie`

change
> is process at the tie
to
> is processed at a time

richlv commented:
Sure, added (hopefully this was done correctly).
I also changed several occurrences of "per regions" to "per region".

I'd might further nitpick on comment formatting and consistency - for example, adding a space after # and making them all start with upper or lower case, currently it's inconsistent (like a comment "No argument given with -da option" being followed by "use the argument given with -da" a few lines later) - but that seems out of scope for typo fixes.
